index.js

```diff
@@ -140,6 +140,22 @@ var diff = function(input, output, path=[], options) {
 		return [op];
 	}
 
+	// If input or output is null, we need to delete it then add new data.
+	if ((input === null || output === null) && input !== output) {
```

Comment:
```suggestion
	if (input === null || output === null) {
```

Is there a reason for `input !== output`? As far as I can tell, it serves no purpose; the tests pass either way, and `jsondiff(null, null)` gives 0 ops whether it's there or not.

Reply:
I don't think it's required, as that case is handled on line 115 https://github.com/kbadk/json0-ot-diff/pull/25/files#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346R115

Reply:
Nice suggested change!

Reply:
Thanks! Ah, good point. I suppose `Array.isArray(input) === Array.isArray(output)` isn't even needed on L115, as `deep-equal` hopefully takes care of that, too.

Reply:
Ah yes. There is a story behind that line.

I added that `Array.isArray(input) === Array.isArray(output)` check because `deep-equal` actuall does _not_ take care of that check, at least at the version of it we are using. And, we are sticking with an older version because the newer version broke the browser build and is very bloated. See https://github.com/kbadk/json0-ot-diff/issues/15

Reply:
Also related https://github.com/inspect-js/node-deep-equal/issues/97

We might want to consider migrating to https://www.npmjs.com/package/fast-deep-equal , in which case we could probably remove that `Array.isArray(input) === Array.isArray(output)` check.

Reply:
There was, no need for the check. Thought it was necessary, didn't check if other code covers that case. Nice catch!

Reply:
Thanks for fixing it!